tests/sources/test_jira.py

```diff
@@ -261,22 +292,6 @@
 ACCESS_CONTROL = "_allow_access_control"
 
 
-@asynccontextmanager
-async def create_jira_source():
-    async with create_source(
-        JiraDataSource,
-        data_source="jira_cloud",
-        username="admin",
-        password="changeme",
-        account_email="me@example.com",
-        api_token="abc#123",
-        jira_url="http://127.0.0.1:8080",
-        projects="*",
-        ssl_enabled=False,
-    ) as source:
-        yield source
-
-
```

Comment:
Please re-implement this wrapper. This is how we are trying to set up unit test configurations going forward now that we don't set any default `value` fields.

Reply:
Got it!